### DISCUSSION TITLE
Hard button to power off without asking

### DISCUSSION THREAD

**Discussion by zachari043** [elementary/triage#256] [Ideas]
I understand the reasons not to power off directly but it's a point of view... This is why settings were invented, in my case I power off the computer automatically with Tuya from my phone, but in elementary I cant... Please add the option, or to please the two point of view add a countdown to the switch off dialogue ;)

**Comment by cosmic-zip** [comment]
Maybe, bind an short cut to shutdown -h now command solve you problem.
see: https://linuxhandbook.com/linux-shutdown-command
